src/utils/helpers.js

```diff
@@ -14,6 +14,23 @@ export const parseInputErrors = (error) => {
   return error;
 };
 
+export const parseApiErrors = (error) => {
```

Comment:
I added that scenario, because the errors coming from the API have the following format:

`errors: {
    user: ["list", "of", "errors"],
    target: ["list", "of", "errors"]
}`

Maybe I should add this to the original `parseInputError` function, which is used for the Form's Input. But I thought the API errors should be parsed in a function independently. 

Thoughts @aguscha333 ?

Reply:
We're going to start using code analysis tools, so having this function repeated will be triggered as code duplication. Maybe we can rename the `parseInputError` function on the base and use it for the API errors as well because it does exactly the same. I would do the same change in this specific project also

Reply:
Why do you think it's necessary to parse objects here?
The structure that the backend sends its the structure that redux-form is expecting actually. If you use `trow SubmissionError(errors)` with errors being the object you should automatically see the errors displayed in the inputs.
What you have to take into account is when the `parseInputErrors` is being called and what is actually receiving.
if you have the following errors structure:
`{ _error: ["list", "of", "errors"], user: ["list", "of", "errors", "for", "user"], target: ["list", "of", "errors", "for", "target"] }`

The method is being called in each input and for global errors, so for the input `user` you already receive just `["list", "of", "errors", "for", "user"]` and for the global error you just receive `["list", "of", "global", "errors"]`

You have to be careful when parsing objects of errors just like that because sometimes forms have more complex structures with [`FormSection`](https://redux-form.com/6.7.0/docs/api/formsection.md/) and [`FormArray`](https://redux-form.com/7.4.2/docs/api/fieldarray.md/). You will end up with an object with nested objects and that is the correct structure. This is actually used a lot in the project I am currently working on.

If you have a case where you are not using redux form that requires you to parse an object as you are doing here I would treat that as a separate case but I don't think it's necessary to add it to the base.